tests/withdrawals_test.go

```diff
@@ -0,0 +1,71 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWithdrawalFieldsInBlocks(t *testing.T) {
+	requireBase := require.New(t)
```

Comment:
nit: I had this problem as well, where a new `require` object needs to instantiated in a nested scope with a nested testing.T. 
I did not like the solution of renaming the variable because it pollutes the name (it is no longer `expect.Something`) 
the require package implements all the symbols found in the require object in the main namespace as well:
https://pkg.go.dev/github.com/stretchr/testify/require#Equal

By writing `require.NotError(t, err)`  it may get a little more readable in cases where nesting is needed.

Reply:
mmmmm I see, so can probably just instantiate one require and pass the `t` when running it in subtests. I will keep in mind for future work. thanks !!!